content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -1,11 +1,27 @@
 +++
 title = "Installation"
-description = "Options for deploying Kubeflow Pipelines"
 weight = 1
 +++
 
 {{% kfp-v2-keywords %}}
 
-This page will be available soon. For similar information, see [KFP v1 installation documentation][v1-installation].
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
+[Kubeflow deployment](/docs/started/#installing-kubeflow), you also have a choice
```

Comment:
What is "Kubeflow deployment", this is not a defined term.

I assume you mean "Kubeflow Platform".

Also, this link is wrong.

Reply:
> What is "Kubeflow deployment", this is not a defined term.
> 
> I assume you mean "Kubeflow Platform".

As requested in the issue, which BTW is open since April and was planned for Kubeflow 1.9 by the release team, this was copied from https://github.com/kubeflow/website/blob/37c939d436b53836629898ce8be2f7715ab4f8cc/content/en/docs/components/pipelines/legacy-v1/installation/standalone-deployment.md?plain=1#L14
Feel free to send a PR to improve both pages.